src/EnergyPlus/DualDuct.hh

```diff
@@ -77,125 +73,84 @@ namespace DualDuct {
+        Real64 ZoneMinAirFrac = 0.0;    // Fraction of supply air used as current minimum flow
+        Real64 ColdAirDamperPosition = 0.0;
+        Real64 HotAirDamperPosition = 0.0;
+        int OAInletNodeNum = 0;                                    // Alternate Node for VAV:OutdoorAir for Outdoor Air
+        int RecircAirInletNodeNum = 0;                             // Alternate Node for VAV:OutdoorAir for Recirc Air
+        bool RecircIsUsed = true;                                  // if true. then not using recirc duct, which is okay
+        Real64 DesignOAFlowRate = 0.0;                             // Terminal Outdoor Air Design Flow Rate for VAV:OutdoorAir, m3/s
+        Real64 DesignRecircFlowRate = 0.0;                         // Terminal Recirc Air Design Flow Rate for VAV:OutdoorAir, m3/s
+        Real64 RecircAirDamperPosition = 0.0;                      // Alternate Damper Pos Output for VAV:OutdoorAir for Recirc Air
+        Real64 OADamperPosition = 0.0;                             // Alternate Damper Pos Output for VAV:OutdoorAir for Recirc Air
+        Real64 OAFraction = 0.0;                                   // Outdoor Air Fraction for VAV:OutdoorAir
+        int ADUNum = 0;                                            // index of corresponding air distribution unit ZoneHVAC:AirdistributionUnit
+        int CtrlZoneNum = 0;                                       // Pointer to CtrlZone data structure
+        int CtrlZoneInNodeIndex = 0;                               // which controlled zone inlet node number corresponds with this unit
+        int ActualZoneNum = 0;                                     // Pointer to Zone data Structure
+        Real64 OutdoorAirFlowRate = 0.0;                           // report variable for TU outdoor air flow rate
+        bool NoOAFlowInputFromUser = true;                         // avoids OA calculation if no input specified by user
+        int OARequirementsPtr = 0;                                 // - Index to DesignSpecification:OutdoorAir object
+        PerPersonMode OAPerPersonMode = PerPersonMode::ModeNotSet; // mode for how per person rates are determined, DCV or design.
+        int AirLoopNum = 0;                                        // index to airloop that this terminal unit is connected to
+        int ZoneTurndownMinAirFracSchPtr = 0;                      // pointer to the schedule for turndown minimum airflow fraction
+        Real64 ZoneTurndownMinAirFrac = 1.0;         // turndown minimum airflow fraction value, multiplier of zone design minimum air flow
+        bool ZoneTurndownMinAirFracSchExist = false; // if true, if zone turndown min air frac schedule exist
+        bool MyEnvrnFlag = true;                     // environment flag
+        bool MySizeFlag = true;                      // sizing flag
+        bool MyAirLoopFlag = true;                   // airloop flag
+        bool CheckEquipName = true;
```

Comment:
Initialization values at the declaration site, so much easier to track.

Reply:
Certainly this is much better than those gnarly unreadable default constructors.